rumi/msg_rumi/reporter.py

```diff
@@ -219,7 +223,45 @@ def print_stats(self, stats):
             )
         )
 
-    def print_details(self, details):
+        if dump_path != "":
+            with open(Path(dump_path) / "translation_stats.json", "w+") as outfile:
+                json.dump(stats, outfile, indent=4)
+
+    def print_score(self, stats):
+        """
+        Print a total score of translation coverage.
+        Parameters
+        ----------
+        stats: dictionary
+            {
+                locale: {
+                    "total": int,
+                    "open": int,
+                    "updated": int,
+                    "completed": int
+                }
+            }
+        Returns
+        -------
+        score: float
+            Score of translation coverage averaging over each target file.
+        """
+        n_completed, n_total = 0, 0
+
+        for locale in stats:
+            stat = stats[locale]
+            # To exclude source files (the original content to be translated).
+            # Source files show 0 in "open", "updated", and "completed"
+            if not stat["open"] + stat["updated"] + stat["completed"] == 0:
```

Comment:
```suggestion
            if stat["open"] + stat["updated"] + stat["completed"] != 0:
```

Reply:
This will work as is, but I think the common convention is to use `!=` here because this is more of a mathematical expression than a logical one.